src/kit/LineChart.tsx

```diff
@@ -26,14 +26,17 @@ const getScientificNotationTickValues: uPlot.Axis['values'] = (_self, rawValues)
   const useNotation = !!rawValues.find(
     (val) => val > 9_999 || val < -9_999 || (0 < val && val < 0.0001) || (-0.0001 < val && val < 0),
   );
-  return useNotation ? rawValues.map((val) => (val === 0 ? val : val.toExponential(2))) : rawValues;
+  return useNotation
+    ? rawValues.map((val) => (val === 0 ? val : val.toExponential(2)))
```

Comment:
question: why 0 is an exception?

Reply:
Mostly because this is based on our original scientific notation function, but also `0.00e+0` sticks out as odd to me

Reply:
not sure which one is the best one, but i think we should follow the standard.